go/state/state_db.go

```diff
@@ -52,6 +52,7 @@ type VmStateDB interface {
 	SetState(common.Address, common.Key, common.Value)
 	GetTransientState(common.Address, common.Key) common.Value
 	SetTransientState(common.Address, common.Key, common.Value)
+	HasEmptyStorage(common.Address) bool
```

Comment:
"HasEmptyStorage" => "HasEmptyState"; also, move it 2 lines up, before the transient state functions.